pkgs/tools/misc/mlxbf-bootctl/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mlxbf-bootctl";
+  version = "1.1-6";
+
+  src = fetchFromGitHub {
+    owner = "Mellanox";
+    repo = pname;
+    rev = "${pname}-${version}";
+    hash = "sha256-F49ZZtty+NARXA/doAFLhsQn4XkPW6GWLXGy4waIaM0=";
+  };
+
+  installPhase = ''
+    mkdir -p "$out"/bin
+    cp -a mlxbf-bootctl "$out"/bin/
+  '';
+
+  meta = with lib; {
+    description = "Control BlueField boot partitions";
+    homepage = "https://github.com/Mellanox/mlxbf-bootctl";
```

Comment:
please put a changelog too :) (I have to ensure you get better quality packages)

Reply:
`https://github.com/Mellanox/mlxbf-bootctl/tags` might be enough